sigstore/_cli.py

```diff
@@ -655,7 +725,7 @@ def _sign(args: argparse.Namespace) -> None:
         print(f"Transparency log entry created at index: {result.log_entry.log_index}")
 
         sig_output: TextIO
-        if outputs["sig"]:
+        if outputs["sig"] in outputs:
```

Comment:
This is a bug: the value `outputs["sig"]` will never be `in outputs` (the set of keys in `outputs`), and thus it will never write the `.sig` file.

I think this was meant to be:

```suggestion
        if outputs["sig"] is not None:
```

(I'm also curious why this wasn't caught by a test.)

Reply:
Yeah, this is supposed to be `"sig" in outputs` since we don't actually know the `"sig"` key is even present; this looks like a bad bulk edit on my part.

(I think we didn't catch this because the CLI code still isn't well-covered. But this is a good incentive to begin to raise it 🙂)

Reply:
d045d21

Reply:
(I looked at our tests again, and this would have been caught by the conformance suite once it merged on main -- this is making me think that we should probably also run the conformance suite on PRs.)